applications/StructuralMechanicsApplication/tests/test_adjoint_loading_conditions.py

```diff
@@ -117,7 +117,7 @@ def _LineLoadCondition3D2N(self, prefix = ""):
 
         # check w.r.t. to design variable SHAPE_SENSITIVITY
         reference_res_2 = ((0,2499.999376,2499.999376,0,2499.999376,2499.999376),(0,2499.999376,2499.999376,0,2499.999376,2499.999376),
-        (0,-0.001250,-0.001250,0,-0.0012450,-0.001250),(0,-2500.000624,-2500.000624,0,-2500.000624,-2500.000624),
+        (0,-0.001250,-0.001250,0,-0.001250,-0.001250),(0,-2500.000624,-2500.000624,0,-2500.000624,-2500.000624),
         (0,-2500.000624,-2500.000624,0,-2500.000624,-2500.000624),(0,-0.001250,-0.001250,0,-0.001250,-0.001250))
         cond.CalculateSensitivityMatrix(KratosMultiphysics.SHAPE_SENSITIVITY, sen_matrix, mp.ProcessInfo)
         self.__CheckSensitivityMatrix(sen_matrix, reference_res_2)
```

Comment:
please use the one for `KratosUnittest`

Reply:
I tried it, but then `refernce_res_2` needs to be from a different data type i.e. each matrix component needs to be defined like "matrix[1,1] = x". Since there are so many entries I decided to implement a custom check function instead. Or is there a better way to define a matrix which I don't know?

Reply:
For solving this properly the `Kratos::Matrix` would need a constructor that accepts e.g. a `list` of `list`s (aka a matrix)
You could try to do this, it would be in this file: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_matrix_to_python.cpp#L74